### PR TITLE
Fix container composes w/r/t multistage container builds

### LINKED ISSUE #11512: Inital Docker container setup failing
**Describe the bug**
When running the container setup command as described in the [docs](https://docs.forem.com/installation/containers/#installing-forem-using-containers), `bin/container-setup`, the yaml is failing to validate for Docker and errors causing an exit of the command.

**To Reproduce**
1. In your terminal run the following command:
```sh
bin/container-setup
```

**Expected behavior**
Docker containers are fully built and initial setup completes successfully

**Screenshots**
image

**Desktop (please complete the following information):**

- OS, version: macOS 11.0.1
- Browser, version: n/a

**Smartphone (please complete the following information):**

- Device: n/a
- OS, version: n/a
- Browser, version: n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by jdoss**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR will use premade images for container composes. Instead of building the containers on the workstations locally, let's just use the premade containers to save time. We mount most of the code in via volumes so wasting time building the containers locally really isn't worth the time. 

Fixes #11512

## [optional] What gif best describes this PR or how it makes you feel?

https://www.youtube.com/watch?v=vabnZ9-ex7o

### CODE REVIEW

**Comment by jdoss** [comment]
Fixes #11512

**Comment by jdoss** [comment]
Please check out #11531 and see if this fixes this problem.

File: .gitignore [hunk 1]
```diff
@@ -82,5 +82,8 @@ db_data
 es_data
 gem_cache
 
+# Compose generated files
+bundle_finished
```
Comment (citizen428) [inline .gitignore:86]: We can leave this as is, but I generally make my sentinel files dotfiles, so they are "hidden".

**Review by citizen428 — APPROVED** [review]

**Review by rhymes — APPROVED** [review]
I was able to boot now. 

Agree with @citizen428 on using a dotfile for `bundle finished`

File: .gitignore (hunk 1 above)
Comment (jdoss) [reply .gitignore:86]: Nice point. Updated to a new path with a hidden file.

**Review by mstruve — APPROVED** [review]
Nice! 🔥

**Pull request merged**